### ISSUE TITLE
Implement Telemetry

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/mozilla-vpn-client#8041]
### User Stories
* As a Product Manager … I want to understand how often users on Monthly plans display interest in upgrading to an Annual plan by virtue of using the in-app Profile screen CTA.
* As a Product Manager … I want to understand how often users who come from the in-app Profile screen CTA end up upgrading to an Annual Plan.

### Requirements
* The CTA in the in-app profile view should be tracked in accordance to our telemetry naming conventions (ask Santiago or Yeonjoo upon doubt).
* The CTA in the in-app profile view should be tagged with appropriate Google Analytics UTM parameters that help us understand upgrades originated in the profile view CTA.

- - -
Miro board: [https://miro.com/app/board/uXjVNfZs_pQ=/?share_link_id=58170517740](https://miro.com/app/board/uXjVNfZs_pQ=/?share_link_id=58170517740|smart-link)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/VPN-5547)
┆Reporter: Santiago Andrigo

**Comment by data-sync-user** [comment]
➤ Santiago Andrigo commented:

Lesley Norton FYI - telemetry spec updated!

**Comment by data-sync-user** [comment]
➤ Matt Lichtenstein commented:

“Sign out” telemetry from the miro board is not included in this ticket because we don’t currently have a sign out button on the subscription management view. This will be coming by way of https://mozilla-hub.atlassian.net/browse/VPN-5696 ( https://mozilla-hub.atlassian.net/browse/VPN-5696|smart-link )

**Comment by data-sync-user** [comment]
➤ Valentina Virlics commented:

We will verify this across platforms after we have macOS builds as well - https://mozilla-hub.atlassian.net/browse/VPN-6054 ( https://mozilla-hub.atlassian.net/browse/VPN-6054|smart-link ).

**Comment by data-sync-user** [comment]
➤ Valentina Virlics commented:

We verified this on Windows and Linux while using last 2.20.0 (2.202312212227) build.

We noted the results in the following document ( https://docs.google.com/spreadsheets/d/1QzGpdiGqamU9jdV0agdexjrbWLiXnOFP7gRY5EG3I5A/edit#gid=1977572129 );

As mentioned below, we will finalize this verification when macOS build will be available.

**Comment by data-sync-user** [comment]
➤ Valentina Virlics commented:

The https://mozilla-hub.atlassian.net/browse/VPN-6054 ( https://mozilla-hub.atlassian.net/browse/VPN-6054|smart-link ) fix was merged and we were able to verify this on macOS platform as well.

The events are displayed, the only thing that raises a question is related to the event triggered when accessing Profile screen.

Old event/category is: 

!image-20240103-130827.png|width=231,height=97!

Implementation:

!image-20240103-130904.png|width=333,height=120!

Current event/category:

!image-20240103-130935.png|width=1155,height=64!

Matt Lichtenstein Is this correct? Thanks!
